site/content/contribute/mobile/build-your-own/android.md

```diff
@@ -0,0 +1,48 @@
+---
+title: "Build the Android App"
+date: 2018-05-20T11:35:32-04:00
+weight: 4
+subsection: Build Your Own App
+---
+
+Android requires that all apps be digitally signed with a certificate before they can be installed, so to distribute your Android application via the Google Play Store, you'll need to generate a signed release APK.
+
+#### Generating a Signing Key
+
+To generate the signed key, we'll be using **keytool** which comes with the JDK required to develop for Android.
```

Comment:
Re-word to:

To generate the signed key, use **keytool** which comes with the JDK required to develop the Android app.